eng/pipelines/templates/jobs/sdk-diff-tests.yml

```diff
@@ -80,6 +80,21 @@ jobs:
       allowFailedBuilds: true
       downloadPath: $(Pipeline.Workspace)/Artifacts
       checkDownloadedFiles: true
+
+  - task: DownloadPipelineArtifact@2
+    displayName: Download Artifacts

```

Comment:
Should [```Source Build SDK```](https://github.com/dotnet/installer/pull/17233/files#diff-066ab785d64a93dd6f7a386327f1d40f22b0e604040308ac157a8ba482ac10c2R70) be changed to ```Source Built SDK``` as well?

Reply:
I don't think it's necessary to make that change here. "Build" and "Built" are fairly interchangeable. I suggested "Built" because that's the term used in the actual filename.